### ISSUE TITLE
Throw out the FIDO counter

### ISSUE THREAD

**Issue by My1** [mooltipass/minible#378]
### Missing feature

I wouldnt call it a missing feature but rather a feature that is uneeded and complicates it in an unneeded fashion.

Basically I would propose to just kick out the FIDO counter for newly made databases (or databases that dont have any fido creds yet) via some sort of flag that could be stored inside the database

https://www.w3.org/TR/webauthn-2/#sctn-op-make-cred step 10
and
https://www.w3.org/TR/webauthn-2/#sctn-op-get-assertion step 9
allow the counter to just stay at zero all the time

So I would propose that in some future enhancement new databases or new credentials, whatever is most appropriate, come with a non-incrementing counter of zero.

### Justification
The assertion the counter gives is that a FIDO device has not been cloned, but with the Mooltipass (and also all the cryptocoin wallets that support FIDO as well) that assertion does not fly as their point is to be backed up and restored perhaps on another device.

FIDO is made to work without the need of drivers or otherwise specific things you need to install because the OS and browsers already support it in general. Needing Moolticute on every fresh start of the device to set the time seems counterintuitive to the entire premise

### Workarounds
basically always have a device with moolticute ready or to never shut down the device, neither of which are great options.
